tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -222,6 +222,9 @@ struct Writer {
 
   subarrayRanges @4 :Subarray;
   # The query subarray/ranges object, new style range object
+
+  varOffsetsMode @5 :Text;
```

Comment:
I know this will be an annoying request, but it looks like we will share all three of these related flags in both `QueryReader` and `QueryWriter`. 

Both of these structs are encapsulated in the `Query` struct. Let's move put `varOffsetsMode`, `varOffsetsAddExtraElement`, and `varOffsetsBitsize` in `struct Query`. This will require changes in `serialization/query.cc`. You can do this in a separate patch.

As long as this is done before we cut the next release (at least a week away), we do not need to worry about backwards compatibility.

Reply:
I included these changes in the current patch, please check if it is what you expected.